src/bench/coin_selection.cpp

```diff
@@ -42,21 +42,19 @@ static void CoinSelection(benchmark::Bench& bench)
     }
     addCoin(3 * COIN, wallet, wtxs);
 
-    // Create groups
-    std::vector<OutputGroup> groups;
+    // Create coins
```

Comment:
in 6148a8acda5e594bb9b3b2d989056f9e03ddbdbd

nit: This comment isn't that helpful, I would suggest something like "Prepare coins in a format that can be passed to SelectCoinsMinConf()"